samza-core/src/main/scala/org/apache/samza/config/ShellCommandConfig.scala

```diff
@@ -25,6 +25,11 @@ object ShellCommandConfig {
    */
   val ENV_COORDINATOR_SYSTEM_CONFIG = "SAMZA_COORDINATOR_SYSTEM_CONFIG"
 
+  /**
+   * This enviorment variable is used to pass configs provided during job submission.
```

Comment:
1. s/enviorment/environment
2. Is this a JSON serialized map like `ENV_COORDINATOR_SYSTEM_CONFIG`? If so, could you please specify that in the javadoc?

Reply:
/ whatever the format of the config is, it should be indicated in the javadoc

Reply:
Updated the doc to reflect that this is a JSON serialized map.